src/TTrigger.cpp

```diff
@@ -988,6 +1000,19 @@ bool TTrigger::match(char* subject, const QString& toMatch, int line, int posOff
             return true;
         }
 
+        if (conditionMet && mExpiryCount > -1) {
+            mExpiryCount--;
+
+            if (mExpiryCount == 0) {
+                mpHost->getTriggerUnit()->markCleanup(this);
+
+                if (mudlet::debugMode) {
+                    TDebug(QColor(Qt::yellow), QColor(Qt::darkMagenta)) << tr("Trigger name=%1 expired.\n").arg(mName) >> 0;
+                }
+            } else if (mudlet::debugMode) {
+                    TDebug(QColor(Qt::yellow), QColor(Qt::darkMagenta)) << tr("Trigger name=%1 has %n match(es) left.\n", "", mExpiryCount).arg(mName) >> 0;
```

Comment:
This message might be confused with the Lua variable `matches`, but I'm not sure on how to word it better.

Reply:
Indeed try not using "matches"... Here are some alternative ideas:
* Trigger name=%1 will trigger %n more time(s).
* Trigger name=%1 will fire %n more time(s).
* Trigger name=%1 will work %n more time(s).
* Trigger name=%1 will execute %n more time(s).
* %1 will trigger %n more time(s).

Also how to understand where name %1 ends and normal text continues? Maybe put %1 in quotes?
* Trigger '%1' will trigger %n more time(s).

Reply:
I don't think the trigger name will be confused with the context - thanks for the suggestions, will go with "fire" since we already use in "fire length".